src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -119,14 +119,11 @@ public function __toString()
     /**
      * Creates a Cookie instance from a Set-Cookie header value.
      *
-     * @param string      $cookie A Set-Cookie header value
-     * @param string|null $url    The base URL
-     *
      * @return static
      *
      * @throws \InvalidArgumentException
      */
-    public static function fromString($cookie, $url = null)
+    public static function fromString(string $cookie, string $url = null)
```

Comment:
Since the `$url` parameter accepts `null`, the correct typehint here should be `?string`

```suggestion
    public static function fromString(string $cookie, ?string $url = null)
```

Reply:
we don't add null type-hint to this kind of null to follow the symfony CS